src/utils.ts

```diff
@@ -1,7 +1,30 @@
-// Util to normalize windows paths to posix
-export function normalizeWindowsPath (input: string = '') {
-  if (!input.includes('\\')) {
-    return input
+import { basename } from 'pathe'
+
+export function resolveAliases (_aliases: Record<string, string>) {
+  // Sort aliases from specific to general (ie. fs/promises before fs)
+  const aliases = Object.fromEntries(Object.entries(_aliases).sort(([a], [b]) =>
+    (b.split('/').length - a.split('/').length) || (b.length - a.length)
```

Comment:
We could refactor this with the internal (or even exposed) `comparePaths` utility.